tests/test_crash_reports.py

```diff
@@ -95,6 +97,8 @@ def test_that_top_crashers_reports_links_work(self, mozwebqa, product):
             csp.wait_for_page_to_load()
 
     @pytest.mark.nondestructive
+    @pytest.mark.xfail("'allizom.org' in config.getvalue('base_url')",
+                       reason="S3 bucket is populating with crash data")
```

Comment:
Unfortunately there is no bug related with this. The problem was discovered and remedied by the devs before a bug could be filed. This could resolve itself at any point. I agree, I don't love this, I'm option to suggestions.

An email was sent out to the socorro team:

      It used to be called `org.allizom.crash-stats.rhelmer-test.crashes` but those days are over.
      Now it's called `org-allizom-crash-stats-crashes`

      We migrated something like 90+% of crashes over to the new bucket but in the space between doing that and 
      changing consul, for about a week recent crashes have ended up in the old rhelmer bucket. At the time of writing
      the UUIDs on stage for any signature report most will lead to a "Crash Not Found 404" error.

    So update your middleware and your submitters or whatever applies.